apps/web/components/getting-started/steps-views/UserProfile.tsx

```diff
@@ -96,16 +97,19 @@ const UserProfile = () => {
     },
   ];
 
+  const organization =
+    user.organization && user.organization.id
+      ? {
+          ...(user.organization as Ensure<typeof user.organization, "id">),
+          slug: user.organization.slug || null,
+          requestedSlug: user.organization.metadata?.requestedSlug || null,
+        }
```

Comment:
Due to the way user.organization is set, all it's props are optional. So, handling it like this so that we can have all it's props set correctly